static/js/popovers.js

```diff
@@ -221,9 +222,11 @@ function render_user_info_popover(
         can_set_away,
         can_mute: muting_allowed && !is_muted,
         can_manage_user: page_params.is_admin && !is_me,
+        can_send_private_message:
+            is_active && !is_me && page_params.realm_private_message_policy !== 2,
```

Comment:
We never want to write checks against magic values like 2; I'll just fix this before merging.